Sources/TokamakCore/Views/List.swift

```diff
@@ -0,0 +1,329 @@
+// Copyright 2018-2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 7/2/20.
+//
+
+public struct List<SelectionValue, Content>: View
+  where SelectionValue: Hashable, Content: View {
+  public enum _Selection {
+    case one(Binding<SelectionValue?>?)
+    case many(Binding<Set<SelectionValue>>?)
+  }
+
+  var selection: _Selection
+  let content: Content
+
+  public init(selection: Binding<Set<SelectionValue>>?, @ViewBuilder content: () -> Content) {
+    self.selection = .many(selection)
+    self.content = content()
+  }
+
+  public init(selection: Binding<SelectionValue?>?, @ViewBuilder content: () -> Content) {
+    self.selection = .one(selection)
+    self.content = content()
+  }
+
+  public var body: some View {
+    ScrollView {
+      HStack {
+        Spacer()
+      }
+      VStack(alignment: .leading) { () -> AnyView in
+        if let contentContainer = content as? ParentView {
+          return AnyView(ForEach(Array(contentContainer.children.enumerated()), id: \.offset) { _, view in
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/List.swift;
  Line: 45;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 106 characters (`line_length`)